src/device/pf_cmina.c

```diff
@@ -762,6 +771,20 @@ static const char *pf_cmina_state_to_string(
    return s;
 }
 
+/**
+ * @internal
+ * Print an IPv4 address (without newline)
+ *
+ * @param ip      In: IP address
+*/
+void pf_ip_address_show(uint32_t ip){
+   printf("%"PRIxLEAST32".%"PRIxLEAST32".%"PRIxLEAST32".%"PRIxLEAST32,
```

Comment:
Wouldn't PRIx8 be more correct here?

Reply:
ip is a 32-bit integer. I'm getting this compilation error for rt-kernel:
```
/home/jonas/projects/profinetstack/p-net/src/device/pf_cmina.c:781:11: error: format '%hhx' expects argument of type 'int', but argument 2 has type 'long unsigned int' [-Werror=format=]
    printf("%"PRIx8".%"PRIx8".%"PRIx8".%"PRIx8,
           ^~~
       ((ip >> 24) & 0xFF),
       ~~~~~~~~~~~~~~~~~~~

```

Reply:
Hmm, that's unexpected. I tested with a generic arm-none-eabi-gcc and see the same issue, but it works for x64-gcc. Not sure why. PRIx32 works and is maybe a bit cleaner. Or cast to uint8_t (and maybe avoid inttypes.h), like we do in most other places.

Reply:
Did cast to unsigned